src/clj/centrebull/shooters/core.clj

```diff
@@ -3,4 +3,7 @@
             [ring.util.http-response :as response]))
 
 (defn create! [{:keys [body-params]}]
-  (response/ok (dao/create! body-params)))
\ No newline at end of file
+  (response/ok (dao/create! body-params)))
+
+(defn suggest [{:keys [params]}]
+  (response/ok (dao/suggest (:q params))))
```

Comment:
Jump into IntelliJ settings and force a new line at the end of file. Removes this ascii symbol